kvm/v13.0.0/release.yaml

```
@@ -0,0 +1,56 @@
+# Generated with:
+# devctl release create --name 13.0.0 --base v12.3.2 --releases . --provider kvm --component etcd@3.4.13 --component kvm-operator@3.14.0 --component containerlinux@2512.5.0 --component kubernetes@1.18.12 --component calico@3.15.3 --app kube-state-metrics@1.2.0@1.9.7 --app cert-exporter@1.3.0 --app net-exporter@1.9.2 --app node-exporter@1.7.0@1.0.1 --overwrite true
+apiVersion: release.giantswarm.io/v1alpha1
+kind: Release
+metadata:
+  creationTimestamp: null
+  name: v13.0.0
+spec:
+  apps:
+  - name: cert-exporter
+    version: 1.3.0
+  - name: chart-operator
+    version: 2.5.0
+  - componentVersion: 1.6.5
+    name: coredns
+    version: 1.2.0
+  - componentVersion: 1.9.7
+    name: kube-state-metrics
+    version: 1.3.0
+  - componentVersion: 0.3.6
+    name: metrics-server
+    version: 1.1.1
+  - name: net-exporter
+    version: 1.9.2
+  - componentVersion: 1.0.1
+    name: node-exporter
+    version: 1.7.0
+  components:
+  - name: app-operator
+    version: 2.7.0
+    releaseOperatorDeploy: true
+  - name: calico
+    version: 3.15.3
+  - name: cert-operator
+    releaseOperatorDeploy: true
+    version: 0.1.0
+    reference: 0.1.0-2
+  - name: cluster-operator
+    releaseOperatorDeploy: true
+    version: 0.23.18
```

```suggestion
    version: 0.23.19
```

cluster-operator release is also out now.